dolphinscheduler-ui/package.json

```diff
@@ -19,7 +19,7 @@
     "echarts": "^5.3.3",
     "js-cookie": "^3.0.1",
     "lodash": "^4.17.21",
-    "monaco-editor": "^0.34.0",
+    "monaco-editor": "^0.50.0",
```

Comment:
Please update the license info in `dolphinscheduler-dist/release-docs/LICENSE` and `dolphinscheduler-dist/release-docs/licenses/ui-licenses/LICENSE-monaco-editor`

Reply:
Done.